src/scss/tools/mixins/_type.scss

```diff
@@ -1,5 +1,6 @@
 @use '../../settings/variables' as v;
 @use '../functions/strip-units' as functions;
```

Comment:
Could you change this to `@use '../functions/strip-units'` and then change the references below please, I must have missed this one. The the next line can just be `@use '../functions/units'`

Reply:
Good spot - will get on this shortly!